test/integration/report-validation.test.js

```diff
@@ -34,20 +35,24 @@ const reportTests = [{
 	name: '@web/test-runner',
 	path: './d2l-test-report-web-test-runner.json',
 	expected: testReportV2PartialWebTestRunner
+}, {
+	name: 'webdriverio',
+	path: './d2l-test-report-webdriverio.json',
+	expected: testReportV2PartialWebdriverIO
 }];
 
 describe('report validation', () => {
 	for (const reportTest of reportTests) {
 		describe(reportTest.name, () => {
-			it('schema', () => {
+			it('schema', function() {
 				if (!hasContext()) {
 					new Report(reportTest.path, { context: testContext });
 				} else {
 					new Report(reportTest.path);
 				}
 			});
 
-			it('contents', () => {
+			it('contents', function() {
```

Comment:
Wondering if this was a mistake. Shouldn't need to be `function()` can just be `() =>`

```suggestion
			it('contents', () => {
```